python/src/aiconfig/default_parsers/openai.py

```diff
@@ -493,9 +496,21 @@ def add_prompt_as_message(
     output = aiconfig.get_latest_output(prompt)
     if output:
         if output.output_type == "execute_result":
-            output_message = output.data
-            if output_message["role"] == "assistant":
-                messages.append(output_message)
+            if output.metadata.get("role", "") == "assistant":
+                output_data = output.data
+                if isinstance(output_data, str):
+                    output_message = {
+                        "content": output_data,
+                        "role": "assistant",
```

Comment:
Why is `"role": "assistant"` hardcoded here?

Reply:
All our output chats can only exist if they have the `role: assistant` as defined. This means that the model is the one who "answered"

Code pointer: https://github.com/lastmile-ai/aiconfig/blob/main/python/src/aiconfig/default_parsers/openai.py#L100-L127